packages/sdk/src/sendPanic.ts

```diff
@@ -82,7 +79,7 @@ export async function sendPanic(error: RustPanic, cliVersion: string, engineVers
       dbVersion: dbVersion,
     }
 
-    const signedUrl = await createErrorReport(params)
+    const signedUrl = await exports.createErrorReport(params)
```

Comment:
`exports.createErrorReport` allows me to mock `createErrorReport` from `jest`, to change its implementation to a function that always throws a specific error `error-report-creation-failed`